python/tskit/provenance.py

```diff
@@ -72,6 +80,31 @@ def get_environment(extra_libs=None, include_tskit=True):
     return env
 
 
+def get_resources():
```

Comment:
Do we want to do this here? I feel like this will get misused pretty badly as people won't realise that time is relative to when tskit got imported. It's only really appropriate in cases where the time is essentially the full life of the interpreter.